static/app/views/performance/transactionSummary/utils.tsx

```diff
@@ -22,6 +23,12 @@ export function generateTransactionSummaryRoute({orgSlug}: {orgSlug: String}): s
   return `/organizations/${orgSlug}/performance/summary/`;
 }
 
+function removeTransactionFilters(query: string): string {
```

Comment:
Nit: This function will most likely be expanded upon as we find more things that needs to be removed. Can we rename this to something a little more generic? Maybe `cleanTransactionSummaryFilter`? If you have a better idea, feel free to use it.